### PR TITLE
add skipMlopsTests maven flag to skip python tests during reflex-algo…

### PR DESCRIPTION
**Pull request by yakov-g**
…s build

usage: mvn install -DskipMlopsTests

### CODE REVIEW

File: reflex-algos/pom.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
         <shaded.prefix>com.parallelmachines.shaded</shaded.prefix>
         <scala.binary.version>2.11</scala.binary.version>
         <reflexVersion>1.5.0</reflexVersion>
+        <skipMlopsTests>false</skipMlopsTests>
```
Comment (zohar-mizrahi) [inline reflex-algos/pom.xml]: Maybe `skipPyTests`, because it skips both *mlops* and *mlcomp*

**Review by zohar-mizrahi — APPROVED** [review]

File: reflex-algos/pom.xml (hunk 1 above)
Comment (yakov-g) [reply reflex-algos/pom.xml]: did it differently...
exec=maven-plugin supports -Dexec.skip to skip whole plugin execution.
Expand this to be able to provide -Dexec.skip=test

**Review by liororama — APPROVED** [review]

File: reflex-algos/pom.xml [hunk 2]
```diff
@@ -262,6 +262,41 @@
             </activation>
             <build>
                 <plugins>
+
+                    <!--
+                    This plugin defines runtime flag skipPyTests when user provides -Dexec.skip or -Dexec.skip=pytest
```
Comment (zohar-mizrahi) [inline reflex-algos/pom.xml]: How `exec.skip` is related to python tests skipping? The property seems to be too generic to be referred to the python tests.
BTW - was there any issue with your previous solution? (Defining `skipPyTests` as property)

Comment (yakov-g) [reply reflex-algos/pom.xml]: exec.skip is the flag defined for Maven exec plugin. If set to true all the execution of the plugin will be skipped. In our case it is building all python packages.
So i didn't want to add a new flag, but added a value instead.

**Pull request merged**